addons/project/static/tests/project_notebook_task_list_tests.js

```diff
@@ -0,0 +1,93 @@
+/* @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+import { addModelNamesToFetch } from "@bus/../tests/helpers/model_definitions_helpers";
+
+import { start } from "@mail/../tests/helpers/test_utils";
+import { contains } from "@web/../tests/utils";
+import {
+    click,
+    getFixture,
+} from "@web/../tests/helpers/utils";
+import { setupViewRegistries } from "@web/../tests/views/helpers";
+
+addModelNamesToFetch(["project.project", "project.task"]);
+
+let target;
+
+QUnit.module('Notebook Tasks List tests', {
+    beforeEach: async function () {
+        const pyEnv = await startServer();
+        const projectId = pyEnv['project.project'].create([
+            { name: "Project One" },
+        ]);
+        const userId = pyEnv['res.users'].create([
+            { name: "User One", login: 'one', password: 'one' },
+        ]);
+        pyEnv['project.task'].create([
+            { name: 'task one', project_id: projectId, closed_subtask_count: 1, subtask_count: 4, child_ids: [2, 3, 4, 7], depend_on_ids: [5,6], state: '04_waiting_normal', user_ids: [userId] },
+            { name: 'task two', parent_id: 1, closed_subtask_count: 0, subtask_count: 0, child_ids: [], depend_on_ids: [], state: '03_approved' },
+            { name: 'task three', parent_id: 1, closed_subtask_count: 0, subtask_count: 0, child_ids: [], depend_on_ids: [], state: '02_changes_requested' },
+            { name: 'task four', parent_id: 1, closed_subtask_count: 0, subtask_count: 0, child_ids: [], depend_on_ids: [], state: '1_done' },
+            { name: 'task five', closed_subtask_count: 0, subtask_count: 1, child_ids: [6], depend_on_ids: [], state: '03_approved' },
+            { name: 'task six', parent_id: 5, closed_subtask_count: 0, subtask_count: 0, child_ids: [], depend_on_ids: [], state: '1_canceled' },
+            { name: 'task seven', parent_id: 1, closed_subtask_count: 0, subtask_count: 0, child_ids: [], depend_on_ids: [], state: '01_in_progress', user_ids: [userId] },
```

Comment:
do you need to specify the closed_subtask_count / subtask_count ? are they not computed fields ?